modules/plugin/feature-pregeneralized/pom.xml

```diff
@@ -84,6 +84,12 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>4.5.1</version>
```

Comment:
Do not track library version numbers in the module poms, they should be managed centrally in the top level library pom instead... which it is happening already for mockito, but it's using version 3.x

If you absolutely need to use 5.x for some reason, the whole library needs to be upgraded, including handling eventual API deprecations (no deprecated API usage tolerated, QA check will fail the build).

Oh, just verified it works also with the current Mockito version.

Reply:
Yeah ... got your point .. version numbers are handled in the top level POM ... removing the version from here suffices